sdk/attestation/attestation/test/public/tokenCertTests.spec.ts

```diff
@@ -22,44 +23,68 @@ describe("TokenCertTests", function() {
     await recorder.stop();
   });
 
-  it("#GetCertificatesAAD", async () => {
+  it("#GetCertificateAAD", async() => {
     const client = createRecordedClient("AAD");
-    const signingCertificates = await client.signingCertificates.get();
-    const certs = signingCertificates.keys!;
-    assert(certs.length > 0);
-    for (const key of certs) {
-      assert.isDefined(key.x5C);
-      for (const cert of key.x5C!) {
-        const berCert = Buffer.from(cert, "base64");
-        assert(berCert);
-      }
-    }
+    await getCertificatesTest(client);
```

Comment:
should we inspect the output?

Reply:
We do - that's the code at line 58 and 50.

Reply:
Ops, missed it. Thanks!